src/data_retrieval/__main__.py

```diff
@@ -22,98 +21,188 @@
 parser_mars = subparsers.add_parser("mars", help="ECMWF MARS")
 
 parser_cds.add_argument("type", choices=["ecmwf", "era5"], help="Data types")
+parser_cds.add_argument(
+    "--local", help="Local directory path to save files", type=str
+)
+parser_cds.add_argument(
+    "--upload",
+    help="Flag to upload data to cloud instead of saving locally",
+    action="store_true",
+)
+
+parser_mars = subparsers.add_parser("mars", help="ECMWF MARS")
```

Comment:
this is a bug. The
```python
parser_mars = subparsers.add_parser("mars", help="ECMWF MARS")
```

is already on line 21 and you add it in here again. That messes up the entire CLI parser. Please remove this line